nodes/cma/nodes-cma-autoscaling-custom-trigger-auth.adoc

```diff
@@ -15,50 +15,47 @@ Alternatively, to share credentials between objects in multiple namespaces, you
 
 Trigger authentications and cluster trigger authentication use the same configuration. However, a cluster trigger authentication requires an additional `kind` parameter in the authentication reference of the scaled object.
 
-.Example secret for Basic authentication
+.Example trigger authentication using a bound service account token for Basic authentication
 [source,yaml]
 ----
-apiVersion: v1
-kind: Secret
+kind: TriggerAuthentication
+apiVersion: keda.sh/v1alpha1
 metadata:
-  name: my-basic-secret
-  namespace: default
-data:
-  username: "dXNlcm5hbWU=" <1>
-  password: "cGFzc3dvcmQ="
+  name: secret-triggerauthentication
+  namespace: my-namespace <1>
+spec:
+  boundServiceAccountToken: <2>
+    - parameter: token
+      serviceAccountName: my-service-account <3>
 ----
-<1> User name and password to supply to the trigger authentication. The values in a `data` stanza must be base-64 encoded.
+<1> Specifies the namespace of the object you want to scale.
+<2> Specifies that this trigger authentication uses a bound service account token for authorization when connecting to the metrics endpoint.
+<3> Specifies the name of the service account to use.
 
-.Example trigger authentication using a secret for Basic authentication
+.Example cluster trigger authentication using a bound service account token for Basic authentication
 [source,yaml]
 ----
-kind: TriggerAuthentication
+kind: ClusterTriggerAuthentication
 apiVersion: keda.sh/v1alpha1
 metadata:
-  name: secret-triggerauthentication
-  namespace: my-namespace <1>
+  name: secret-cluster-triggerauthentication <1>
```

Comment:
Should probably change the name to reflect a `boundserviceacccounttoken-triggerauthentication`